src/spyglass/common/common_ripple.py

```diff
@@ -168,44 +190,38 @@ def fetch_dataframe(self):
     @staticmethod
     def get_ripple_lfps_and_position_info(key):
         nwb_file_name = key["nwb_file_name"]
-        interval_list_name = key["interval_list_name"]
+        interval_list_name = key["target_interval_list_name"]
```

Comment:
Since the table is now dependent on an LFP table, which uses "target_interval_list_name" instead of "interval_list_name", so in order to use the key with the LFP tables, "target_" is needed.